Fantasy-Quest/Assets/Scripts/Scene/NewScene/EnterNewScene.cs

```diff
@@ -0,0 +1,95 @@
+using System.Collections;
+using System.Collections.Generic;
+using Common;
+using Interaction.Item;
+using Sirenix.OdinInspector;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Scene
+{
+    [AddComponentMenu("Scripts/Scene/NewScene/Scene.NewScene.EnterNewScene")]
+    internal class EnterNewScene : MonoBehaviour, ISceneTransition
+    {
+        [Required]
```

Comment:
```suggestion
```

Reply:
Use interface for transition in other scripts and inherit Transition.End.Contriller by this interface